src/admin-views/tribe-options-tickets.php

```diff
@@ -247,41 +280,54 @@
 		],
 		'validation_type' => 'options',
 	];
+
+	$misc_fields['tec-settings-general-misc-fields-div-end'] = [
+		'type' => 'html',
+		'html' => '</div>',
+	];
 }
 
 // Authentication requirements fields.
 $auth_fields = [
+	'tec-settings-general-auth-fields-div-start' => [
+		'type' => 'html',
+		'html' => '<div class="tec-settings-form__content-section">',
+	],
 	// Authentication Requirements header.
-	'auth-header'                        => [
+	'auth-header'                                => [
 		'type' => 'html',
-		'html' => '<h3 id="tec-tickets-settings-authentication" class="tec-settings-form__section-header">' .
-			esc_html__( 'Login Requirements', 'event-tickets' ) . '</h3>',
+		'html' => '<h3 id="tec-tickets-settings-authentication" class="tec-settings-form__section-header tec-settings-form__section-header--sub">' .
+					esc_html__( 'Login Requirements', 'event-tickets' ) . '</h3>',
 	],
 	// Authentication Requirements description.
-	'auth-description'                   => [
+	'auth-description'                           => [
 		'type' => 'html',
-		'html' => '<div class="tec-settings-form__description"><p>' .
-			sprintf(
-			// Translators: %1$s: singular RSVP label, %2$s: plural tickets label in lowercase, %3$s: opening link tag to WP general settings, %4$s: closing link tag.
-				_x(
-					'You can require that users log into your site before they are able to %1$s (or buy %2$s). Please review your WordPress Membership option (via the %3$sGeneral Settings admin screen%4$s) before adjusting this setting.',
-					'ticket authentication requirements',
-					'event-tickets'
-				),
-				tribe_get_rsvp_label_singular( 'authentication_requirements_advice' ),
-				tribe_get_ticket_label_plural_lowercase( 'authentication_requirements_advice' ),
-				'<a href="' . esc_url( get_admin_url( null, 'options-general.php' ) ) . '" target="_blank">',
-				'</a>'
-			) .
-			'</p></div>',
+		'html' => '<fieldset><div class="tec-settings-form__description"><p>' .
```

Comment:
This HTML has become overly complicated just to tweak the margins.
The `<fieldset>` removes the top margin and the `<div>` _removed_ the bottom (by making the p last child - which would now be covered if it was just `<fieldset><p>`)

However, a `<fieldset>` with no field in it is _**not**_ semantic. Find another way to do this, even if it means creating a new class.